docs/reference/sql/functions/math.asciidoc

```diff
@@ -276,6 +276,30 @@ include-tagged::{sql-specs}/docs.csv-spec[mathInlinePowerPositive]
 include-tagged::{sql-specs}/docs.csv-spec[mathInlinePowerNegative]
 --------------------------------------------------
 
+[[sql-functions-math-random]]
+===== `RANDOM`
```

Comment:
There should be a `RANDOM` without seed, e.g. `RANDOM()`. If that's the case the documentation should reflect this.

Reply:
Indeed, `RANDOM` should accept no parameters, I missed this. I will look into why this is also not possible at the moment.

Reply:
I will let RANDOM as is for now and deal with no-parameter approach in a separate PR. Can you have another look at the recent commits @costin ?

Reply:
https://github.com/elastic/elasticsearch/issues/34152